

*
* Visual Studio Code Server
*
* Reminder: You may only use this software with Visual Studio family products,
* as described in the license https://aka.ms/vscode-remote/license
*

Extension host agent listening on 44817

[09:59:57] Extension host agent started.
[09:59:57] [127.0.0.1][d0840a33][ManagementConnection] New connection established.
[09:59:57] [127.0.0.1][58f49c90][ExtensionHostConnection] New connection established.
[09:59:57] [127.0.0.1][58f49c90][ExtensionHostConnection] <28829> Launched Extension Host Process.
[10:00:08] [127.0.0.1][d0840a33][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[10:00:08] [127.0.0.1][58f49c90][ExtensionHostConnection] <28829> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[10:00:08] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[10:00:15] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[10:00:15] Cancelling previous shutdown timeout
[10:00:15] [127.0.0.1][8fd55534][ManagementConnection] New connection established.
[10:00:15] [127.0.0.1][b6d9c9be][ExtensionHostConnection] New connection established.
[10:00:15] [127.0.0.1][b6d9c9be][ExtensionHostConnection] <29000> Launched Extension Host Process.
New EH opened, aborting shutdown
[10:05:15] New EH opened, aborting shutdown
IPC "File Watcher (parcel, node.js)" crashed with exit code null and signal SIGSEGV
[10:48:00] [File Watcher (parcel)] restarting watcher after error: terminated by itself with code null, signal: SIGSEGV
[10:50:41] [127.0.0.1][8fd55534][ManagementConnection] The client has reconnected.
rejected promise not handled within 1 second: Error: Could not find pty with id "5"
stack trace: Error: Could not find pty with id "5",    at l._throwIfNoPty (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateIcon (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1651),    at Object.call (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "5",    at l._throwIfNoPty (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateIcon (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1651),    at Object.call (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "5"
stack trace: Error: Could not find pty with id "5",    at l._throwIfNoPty (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateTitle (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1591),    at Object.call (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "5",    at l._throwIfNoPty (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateTitle (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1591),    at Object.call (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[10:50:55] [127.0.0.1][b6d9c9be][ExtensionHostConnection] The client has reconnected.
[11:45:39] [127.0.0.1][8fd55534][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
Error: Unexpected SIGPIPE
    at process.<anonymous> (/root/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
[12:59:43] [127.0.0.1][8fd55534][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[12:59:43] [127.0.0.1][c0d2c5f2][ManagementConnection] New connection established.
[12:59:43] [127.0.0.1][4f33c39b][ExtensionHostConnection] New connection established.
[12:59:43] [127.0.0.1][4f33c39b][ExtensionHostConnection] <6307> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "10": Replaying 4585 chars and 1 size events
[13:04:43] [127.0.0.1][8fd55534][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[13:04:43] [127.0.0.1][b6d9c9be][ExtensionHostConnection] <29000> Extension Host Process exited with code: 0, signal: null.
